src/main/java/de/caritas/cob/userservice/api/model/ConsultantStatus.java

```diff
@@ -0,0 +1,7 @@
+package de.caritas.cob.userservice.api.model;
+
+public enum ConsultantStatus {
```

Comment:
please check usages of get all consultants methods in code, if the existing logic does not have to be amended so that it would filter Consultant based on their Status (currently as I understand status will be just used to display it on the UI, however in our business logic we should probably not process consultant with status ERROR or IN_PROGRESS).

Reply:
so far I was not been able to see any issues here.